test/util/ReduceCssCalcPrototype.spec.ts

```diff
@@ -101,16 +99,7 @@ describe('reduce-css-calc', () => {
           `calc(${capHeight} * -${lineHeight})`,
           `calc(${capHeight} / -${lineHeight})`,
         ].forEach(calcString => {
-          try {
-            const newCalc = reduceCSSCalc(calcString);
-            const prevCalc = oldReduceCSSCalc(calcString);
-
-            if (!(prevCalc.includes('calc') || prevCalc.includes(' '))) {
-              expect(prevCalc).toEqual(newCalc);
-            }
-          } catch (e) {
```

Comment:
Test failures from `expect` are reported as an exception - so this `catch` turns this test into an always green one. Don't catch exceptions in tests.

Reply:
Some of the tests were added to explicitly show that the old reduce css calc threw errors in those cases, but agreed should be maybe been a separate block with expect to throw